apps/central-scan/backend/schema.sql

```diff
@@ -8,7 +8,6 @@ create table election (
   polls_state text not null default "polls_closed_initial",
   ballot_count_when_ballot_bag_last_replaced integer not null default 0,
   is_sound_muted boolean not null default false,
-  cvrs_backed_up_at datetime,
```

Comment:
This value and the corresponding `setCvrsBackedUp` and `getCvrsBackupTimestamp` values haven't been used in a while, as far as I can tell. `scanner_backed_up_at`, on the other hand, is still relevant